index.js

```diff
@@ -9,17 +9,19 @@ const events = {
 	MESSAGE_REACTION_ADD: 'messageReactionAdd',
 	MESSAGE_REACTION_REMOVE: 'messageReactionRemove',
 }
+// Constant for which channels to not allow for deletion
+const PERMANENT_CHANNELS = [823936480056705094, 823651605554921502]
```

Comment:
There has got to be a better way to do this... there are so many more channels (and the Ids change between test server and prod)

Reply:
Those lines don't do anything and should be deleted